src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,377 @@
+be stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data)
+along with code to convert the data to
+SNIRF in the [`/code` directory](../02-common-principles.md#storage-of-derived-datasets).
+The unprocessed raw data should be stored in
+the manufacturer's format before any additional processing or conversion is applied.
+Retaining the native file format is especially valuable in a case when conversion elicits the
+loss of crucial metadata unique to specific manufacturers and fNIRS systems.
+We also encourage users to provide additional meta information extracted from
+the manufacturer-specific data files in the sidecar JSON file. This allows for
+easy searching and indexing of key metadata elements without needing to parse
+the various proprietary (typically binary) native data files. Other relevant
+files should  be included alongside the fNIRS data.
+
+### Terminology
+
+For proper documentation of fNIRS recording metadata, it is important
+to understand the difference between a Source, Detector, and Channel as these
+are defined differently to other modalities, such as EEG. The following definitions
+apply in this document:
+
+-   Source - A light emitting device, sometimes called a transmitter.
+
+-   Detector - A photoelectric transducer, sometimes called a receiver.
+
+-   Optode - Refers to either a source or detector.
+
+-   Channel - A paired coupling of a source and a detector with one specific wavelength of light.
+    It is common for a single Source-Detector pair to result in two or more channels
+    with different wavelengths.
+
+### Sidecar JSON (`*_nirs.json`)
+
+It is common within the fNIRS community for researchers to build their own caps
+and optode holders to position their sources and detectors, or for optodes to
+be directly attached to the scalp with adhesive. To facilitate description of
+the wide variety of possible configurations, several fields are RECOMMENDED within
+the `*_nirs.json` file.
+To clarify the usage and interaction of these fields, the following examples are provided.
+If a commercial cap such as EasyCap actiCAP 64 Ch Standard-2
+was used, then the values `CapManufacturer = ”EasyCap”`, `CapManufacturersModelName = ”actiCAP
+64 Ch Standard-2”` and `NIRSPlacementScheme = ”n/a”` should be used.
+If an EasyCap was used but with custom positions,
+as may be done by cutting custom holes in the cap,
+then the values `CapManufacturer = ”EasyCap”`, `CapManufacturersModelName = ”custom”` and
+`NIRSPlacementScheme = ”n/a”` should be used.
+If a completely custom cap was knitted, then
+`CapManufacturer = ”custom”`, `CapManufacturersModelName = ”custom”` and
+`NIRSPlacementScheme = ”n/a”`.
+If no cap was used and optodes were taped to the scalp
+at positions Cz, C1 and C2, then the values `CapManufacturer = ”none”`, `CapManufacturersModelName
+= ”n/a”` and `NIRSPlacementScheme = ”["Cz", "C1", “C2]”` should be used.
+
```

Comment:
```suggestion
In these cases additional information regarding channels and optodes SHOULD be placed in `*_channels.tsv` and `*_optodes.tsv` files.

```
When there's no info about placement schema, etc, then this `*_channels.tsv` and `*_optodes.tsv` files SHOULD be present, right? or it's just MAY?